src/Form/Type/Organization/AddPackageType.php

```diff
@@ -62,6 +63,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                     new NotNull(),
                 ],
             ])
+            ->add('versionsLimit', IntegerType::class, [
+                'label' => 'Limit number of versions',
```

Comment:
technically, this is not limit, this is number of releases that will be downloaded from source ,and we download them to speed up install process. So maybe `Download latest releases` :thinking: